css/templates/land-see/page.css

```diff
@@ -354,3 +354,115 @@ amp-img.cover img {
 .ampstart-input input[type="email"].user-invalid {
   border: 2px solid var(--color-form-invalid);
 }
+
+/* Accordion */
```

Comment:
can you give this a meaningful name (land-see class). Changing all the accordions is not a good idea, imagine if we add a component later that uses accordion then this template would be broken.

Reply:
Yes, good catch. I name-spaced the other form email input, but not the error handling.